reVX/least_cost_xmission/config/__init__.py

```diff
@@ -4,7 +4,7 @@
 """
 import os
 
-from .xmission_config import XmissionConfig
+from reVX.least_cost_xmission.config.xmission_config import XmissionConfig
```

Comment:
Out of curiosity, why the absolute versus relative import? It seems the relative import is less work if the folder structure changes.

Reply:
No good reason beyond Grant's preference that we started following in reV and has bleed over to reVX. We have run into some strange import issues in the past but I think that was mainly noobness + python 3.6 being weird.